cli/tests/integration/mod.rs

```diff
@@ -1003,6 +1012,7 @@ async fn test_resolve_dns() {
       .unwrap();
     let err = String::from_utf8_lossy(&output.stderr);
     let out = String::from_utf8_lossy(&output.stdout);
+    println!("{}", err);
```

Comment:
Remove

Reply:
removed in 6ccb2ce03ca1ffe3d298dfda028124525d7848a4